exist-core/src/main/java/org/exist/http/servlets/EXistServlet.java

```diff
@@ -177,6 +178,77 @@ protected void doPut(final HttpServletRequest request, final HttpServletResponse
         }
     }
 
+    protected void doPatch(final HttpServletRequest request, final HttpServletResponse response)
+            throws ServletException, IOException {
+        // first, adjust the path
+        String path = adjustPath(request);
+
+        // second, perform descriptor actions
+        final Descriptor descriptor = Descriptor.getDescriptorSingleton();
+        if (descriptor != null) {
+            // TODO: figure out a way to log PATCH requests with
```

Comment:
TODO?

Reply:
Yeah... a _really_ old one

Reply:
Well, actually I added it because it was there for PUT too.

Reply:
But the replay log will be removed anyway, as we discussed yesterday